src/components/data/RecentTracks/RecentTracks.tsx

```diff
@@ -20,11 +22,40 @@ const RecentTracks = (): JSX.Element => {
   return (
     <Stack
       divider={<StackDivider borderColor="gray.700" />}
-      marginTop={50}
+      margin="50px 50px 0 50px"
     >
       <Text fontSize="6xl">
         {t('screens.main.recentTracks')}
       </Text>
+      {currentlyPlayingResponse?.is_playing ? (
+        <Stack
+          direction="row"
+          align="center"
+        >
+          <Image
+            boxSize="60px"
+            src={currentTrack?.album.images[0].url}
+            alt=""
+            mr={8}
+          />
+          <Stack direction="row" maxW="55%">
+            <Link fontSize="lg" href={currentTrack?.artists[0].external_urls.spotify} color="secondary.900" as="b">
+              {currentTrack?.artists[0].name}
+            </Link>
+            <Text fontSize="lg" color="secondary.900">
+              {' - '}
+            </Text>
+            <Link fontSize="lg" href={currentTrack?.external_urls.spotify} color="secondary.900" as="b">
+              {currentTrack?.name}
+            </Link>
+          </Stack>
+          <Text fontSize="lg" pos="absolute" right="5%" maxW="20%" color="secondary.900" as="b">
```

Comment:
jak tu dajesz pos="absolute" to na root powinieneś też dodać pokolenia wyżej pos="relative" (na Stacku) tak żeby to było absolutne w obrębie tego komponentu, nie całego ekranu/strony